resources/metadata/gctsCreateRepository.yaml

```
@@ -0,0 +1,100 @@
+metadata:
+  name: gctsCreateRepository
+  description: Creates a Git repository on an ABAP system
+  longDescription: |
+    Creates a local Git repository on an ABAP system if it does not already exist.
+
+spec:
+  inputs:
+    secrets:
+      - name: credentialsId
+        description: Jenkins credentials ID containing username and password for authentication to the ABAP system on which you want to create the repository
+        type: jenkins
+    params:
+      - name: username
+        type: string
+        description: Username to authenticate to the ABAP system
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        aliases:
+          - name: username
+      - name: password
+        type: string
+        description: Password to authenticate to the ABAP system
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+        mandatory: true
+        aliases:
+          - name: password
+      - name: repositoryName
```

This is typically named as `repository`.

```suggestion
      - name: repository
```